### PR TITLE
Update code for modal window.

### PR DESCRIPTION
**Pull request by lenkaswarna**

### CODE REVIEW

File: src/client/app/app.module.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { APP_BASE_HREF } from '@angular/common';
 import { HttpModule } from '@angular/http';
 import { FormsModule } from '@angular/forms';
 import { ReactiveFormsModule } from '@angular/forms';
+import { DomSanitizer } from '@angular/platform-browser';
```
Comment (arun-awnics) [inline src/client/app/app.module.ts]: Not required.

File: src/client/app/chat/chat.component.ts [hunk 2]
```diff
@@ -264,8 +267,16 @@ export class ChatComponent implements OnInit, AfterViewChecked {
     this.createForm();
     this.receiveMessageFromSocket();
     this.receiveUpdatedMessageFromSocket();
-    //this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+    this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
```
Comment (arun-awnics) [inline src/client/app/chat/chat.component.ts]: Wrong declaration.

Comment (lenkaswarna) [reply src/client/app/chat/chat.component.ts]: this code we are using for appearUrl, its not related to modal window

Comment (arun-awnics) [reply src/client/app/chat/chat.component.ts]: We had a discussion before that you are not going to use domSanitizer in your implementation. Why are you still using it if it is not being implemented?

**Comment by arun-awnics** [comment]
I don't see description for this PR. This cannot be merged unless there is update notes attached. Kindly update the same or create another PR after updating the changes.

File: src/client/app/chat/chat.component.ts (hunk 2 above)
Comment (lenkaswarna) [reply src/client/app/chat/chat.component.ts]: No this one you have implemented for appearUrl .so I haven't removed, for modal window URL, i have removed.

Comment (arun-awnics) [reply src/client/app/chat/chat.component.ts]: Oh... I am sorry. I thought you were trying to use this url in your implementation. How about the other changes I have mentioned?

Comment (lenkaswarna) [reply src/client/app/chat/chat.component.ts]: just removed from app.module.ts.That was the only one.

**Pull request closed without merging**